services/management/backup/locations_service.go

```diff
@@ -130,6 +130,41 @@ func (s *LocationsService) ChangeLocation(ctx context.Context, req *backupv1beta
 	return &backupv1beta1.ChangeLocationResponse{}, nil
 }
 
+// TestLocationConfig tests backup location and credentials.
+func (s *LocationsService) TestLocationConfig(
+	_ context.Context,
+	req *backupv1beta1.TestLocationConfigRequest,
+) (*backupv1beta1.TestLocationConfigResponse, error) {
+	params := &models.TestBackupLocationParams{}
```

Comment:
PMM codebase in general follows this aproach in variable declarations: https://dave.cheney.net/2014/05/24/on-declaring-variables
```suggestion
	var params models.TestBackupLocationParams
```

Reply:
done